web3c/utils.js

```diff
@@ -0,0 +1,16 @@
+/**
+ * objectAssign is an alternative to Object.assign to work with react-native.
+ * Assigns all properties from b onto a.
+ */
+function objectAssign(a, b, enumerable = true) {
+  Object.keys(b).forEach((key) => {
+    Object.defineProperty(a, key, {
+      value: b[key],
+      enumerable,
```

Comment:
Sorry to interject, but I wonder if you might want to set `writable` to `true` as well, just to be safe in case some code down the line wants to modify one of the property values.

Reply:
Good point. I would prefer to keep writable false until we need it (admittedly this doesn't buy us much since much is already writable). @willscott do you have a preference?

Reply:
maybe be clear in the comment that this makes a read-only copy of the object?

Reply:
Addressed this here https://github.com/oasislabs/web3c.js/pull/159